assets/queries/ansible/gcp/high_kms_rotation_period/metadata.json

```diff
@@ -3,7 +3,7 @@
   "queryName": "High KMS Rotation Period",
   "severity": "MEDIUM",
   "category": "Secret Management",
-  "descriptionText": "KMS rotation period should not surpass 365 days",
+  "descriptionText": "KMS Rotation Period should be greater than 365 days.",
```

Comment:
Can you update also in `assets\queries\terraform\gcp\high_kms_rotation_period\metadata.json`, please?

Reply:
done

Reply:
Thank you so much!